README.rst

```diff
@@ -83,6 +83,56 @@ We are defined by defult the following order:
 - `Debug Resources`: When **DRYDOCK_DEBUG** active, over the Sync Wave 100
 - `Horizontal Pod Autoscalers`: When active, over the Sync Wave 150
 
+Steps for migration
+-------------------
+
+This guide delineates the necessary steps for a seamless migration to Olive using Drydock, ensuring a smooth transition with careful consideration of potential challenges.
+
+1. For Olive, it is necessary to update the version of `Drydock <https://github.com/eduNEXT/drydock>`_ to the latest version 15.x.x in the requirements.txt file of your environment, including:
+
+    .. code:: bash
+
+        git+https://github.com/edunext/drydock@v15.x.x#egg=drydock==15.x.x
+
+2. In the `config.yml` file, include variables that activate the initialization jobs and post-migration jobs:
+
+    .. code:: yaml
+
+        DRYDOCK_INIT_JOBS: True
+        DRYDOCK_MIGRATE_FROM: <MAJOR_OF_TUTOR_VERSION>
+
+   Set `DRYDOCK_MIGRATE_FROM` to 13 for migrating from **Maple** or 14 for migrating from **Nutmeg**.
+
+    .. note::
+
+        Regarding initialization jobs:
+
+        Since DryDock version 15.8.1, a mechanism was introduced to automate the inclusion of
+        initialization jobs for both Tutor and Tutor Plugins, ensuring correct execution order.
+        This eliminates the need for manual definition of such jobs.
```

Comment:
```suggestion
        Since Drydock version 15.8.1, a new mechanism was introduced to automatically generate Kubernetes manifest files for the initialization jobs defined by Tutor and Tutor plugins. The generated files are meant to be used by ArgoCD for deployment. Previously you would need to write the manifest files for the initialization jobs manually if you wanted to use a tool like ArgoCD.
```